packages/react-dom/src/client/ReactDOM.js

```diff
@@ -682,9 +671,8 @@ function warnIfReactDOMContainerInDEV(container) {
   if (__DEV__) {
     warningWithoutStack(
       !container._reactRootContainer,
-      'You are calling ReactDOM.%s() on a container that was previously ' +
+      'You are calling ReactDOM.createRoot() on a container that was previously ' +
```

Comment:
I would not expect all fo these conditional names to change, given that we run this test against not-experimental builds as well. Why did they?

Reply:
See the first commit.

> The method names don't get stripped out of the production bundles
because they are passed as arguments to the error decoder.

Slightly unrelated but I did it this way instead of checking `__EXPERIMENTAL__` in the tests. We're probably going to remove the prefixed versions from the stable releases, anyway.